### PR TITLE
npx: add --no-install and --quiet examples

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 8.4.0

### CODE REVIEW

File: pages/common/npx.md [hunk 1]
```diff
@@ -5,12 +5,20 @@
 
 - Execute the binary from a given npm module:
 
-`npx {{module_name}}`
+`npx {{module_name}} {{command_arguments}}`
```
Comment (EmilySeville7cfg) [inline pages/common/npx.md:8]:
```suggestion
`npx {{module_name}} {{arg1 arg2 ...}}`
```
Why not to use `{{arg1 arg2 ...}}`? ;)

File: pages/common/npx.md [hunk 2]
```diff
@@ -5,12 +5,20 @@
...
 
 - In case a package has multiple binaries, specify the package name along with the binary:
 
-`npx -p {{package_name}} {{module_name}}`
+`npx --package {{package_name}} {{module_name}}`
 
-- View help contents:
+- Run a command if existis in the current path or in `node_modules/.bin`:
+
+`npx --no-install {{command}} {{command_arguments}}`
```
Comment (EmilySeville7cfg) [inline pages/common/npx.md:16]:
```suggestion
`npx --no-install {{command}} {{arg1 arg2 ...}}`
```

File: pages/common/npx.md [hunk 3]
```diff
@@ -5,12 +5,20 @@
...
+
+- Execute the binary from a given npm module suppressing any output from `npx` itself:
+
+`npx --quiet {{module_name}} {{command_arguments}}`
```
Comment (EmilySeville7cfg) [inline pages/common/npx.md:20]:
```suggestion
`npx --quiet {{module_name}} {{arg1 arg2 ...}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/npx.md (hunk 1 above)
Comment (sbrl) [reply pages/common/npx.md:8]: `command_arguments` is more generic I think. We should leave this as-is.

**Review by sbrl — APPROVED** [review]

**Pull request merged**